### PR TITLE
Doc on docs

### PR DESCRIPTION
**Pull request by ffesti**
and sync of GH pages header

### CODE REVIEW

File: docs/manual/devel_documentation.md [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+
+## Reference Manual
+
+The [Reference
+Manual](https://rpm-software-management.github.io/rpm/manual/) is
+supposed to cover all user facing aspects of RPM but has still a long
+way to go.
+
+The markdown sources of the Reference Manual are found in
+`docs/manual`. They are not build into html or any other format by the
```
Comment (dmnks) [inline docs/manual/devel_documentation.md:25]: s/build/built/

File: docs/manual/devel_documentation.md [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+make file. Instead the `master` branch is turned in to GitHub
```
Comment (dmnks) [inline docs/manual/devel_documentation.md:26]: Cosmetic, but how about calling it Makefile instead? It's probably a bit more commonly used that way.

File: docs/manual/devel_documentation.md [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+Pages. By sharing the header with the [rpm.org](https://rpm.org) web
+page that links to both it looks like their are one web site.
```
Comment (dmnks) [inline docs/manual/devel_documentation.md:28]: s/their/they're/

File: docs/manual/devel_documentation.md [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+locally. To do that one need to reproduce the relevant infrastructure
+of GitHub Pages - mainly the [Jekyll](https://jekyllrb.com/) static
+website generator. This can be installed from a distribution
+repository (may be named `rubygem-jekyll`) or via the Ruby mechanisms
+e.g. with `gem install jekyll`
+
+GitHub Pages use a number of Jekyll plugins and does more things in the
+background that is not that relevant for our use case. The plugins can
+be found in `docs/_config.yml`. For convenience most are commented
+out. The remaining ones need also to be install (currently
```
Comment (dmnks) [inline docs/manual/devel_documentation.md:40]: s/install/installed/

File: docs/manual/devel_documentation.md [hunk 5]
```diff
@@ -0,0 +1,71 @@
...
+To render the Reference Manual `jekyll build` can be executed in the
+`docs` dir. It will create a `docs/_site` directory with the rendered
+web page - including html versions of the man pages. Alternativly
+`jekyll serve` will open a web server on localhost.
+
+## Man Pages
+
+The sources of the man pages are found in `docs/man/` in markdown
+format and are converted to troff format as used by the `man` tool by
+the make file in this directory (or building the whole of RPM). They
```
Comment (dmnks) [inline docs/manual/devel_documentation.md:58]: Makefile (same as above)

File: docs/manual/devel_documentation.md [hunk 6]
```diff
@@ -0,0 +1,71 @@
...
+are also rendered to HTML together with the Reference Manual.
+
+## Rpm.org
+
+The [rpm.org](https://rpm.org) web page is maintained in a separate
+git project:
+[rpm-web](https://github.com/rpm-software-management/rpm-web). This
+allows storing data like release tar balls that are not acceptable in
+the rpm repository. It is rendered as GitHub Pages and made available
```
Comment (dmnks) [inline docs/manual/devel_documentation.md:67]: Does "it is rendered as GitHub Pages" mean that the html & stylesheet look like GitHub Pages? If so, how about:
"It is rendered in the same way as GitHub Pages..."

**Comment by dmnks** [comment]
Other than my above notes, looks good to me.

File: docs/manual/devel_documentation.md (hunk 6 above)
Comment (ffesti) [reply docs/manual/devel_documentation.md:67]: No, it means what is says:  The git project is rendered and made available by GitHub Pages as GitHub Pages.

File: docs/manual/devel_documentation.md [hunk 7]
```diff
@@ -0,0 +1,71 @@
...
+title: rpm.org - Writing Documentation
+---
+# Writing Documentation
+
+There is a lot of 3rd party documentation on RPM and RPM
+packaging. We collect it on the [Documentation
+page](https://rpm.org/documentation.html) on rpm.org. Many of those
+are unfortunately not quite up-to-date.
+
+Get to an current and continuously updated documentation we are
```
[the remark this answers is not in the document]
Comment (ffesti) [reply docs/manual/devel_documentation.md]: OK, reworded the whole area.

File: docs/manual/devel_documentation.md (hunk 6 above)
Comment (dmnks) [reply docs/manual/devel_documentation.md:68]: Nah of course. OK then

**Comment by dmnks** [comment]
OK, just squash the fix-up commits and we should be good to go.

**Review by dmnks — APPROVED** [review]

**Pull request merged**